src/components/menu/view-controls.jsx

```diff
@@ -26,11 +27,7 @@ export class ViewControlsUnconnected extends React.Component {
 
   render() {
     const loginItem = this.props.isAuthenticated ?
-      (
-        <NotebookTaskButton task={tasks.logoutGithub}>
-          <img src={this.props.avatar} alt="" className="user-avatar" />
-        </NotebookTaskButton>
-      )
+      <EditorAccountMenu avatar={this.props.avatar} />
```

Comment:
Just fyi I am working a bit in this area right now (have some changes I will upload in a moment), so this part of the PR might need to be rebased.